src/posts/keymaps/peterjc-new_stickney.md

```diff
@@ -0,0 +1,24 @@
+---
+author: peterjc
+baseLayouts: ["QWERTY"]
```

Comment:
This is a stretch, but there are no prior Japanese language entries to guide me, and the JIS kana layout is in a sense Qwerty based (the more widely used Romaji Japanese input system is almost always used with Qwerty - but this is an alternative kana entry system).